modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java

```diff
@@ -273,6 +272,31 @@ private Map<String, EntityField> _getStringEntityFieldMap(
 		return entityFieldsMap;
 	}
 
+	private Boolean _isUnsupportedBusinessType(String businessType) {
+		return ListUtil.exists(
+			Arrays.asList(UnsupportedBusinessTypes.values()),
+			unsupportedBusinessType -> unsupportedBusinessType.getValue(
+			).equals(
+				businessType
+			));
+	}
+
 	private final Map<String, EntityField> _entityFieldsMap;
 
+	private enum UnsupportedBusinessTypes {
```

Comment:
I think for this use case, it is not really necessary an Enumeration. What do you think about using a Set of String:

```
private static final Set<String> _unsupportedBusinessTypes =
	SetUtil.fromArray(
		ObjectFieldConstants.BUSINESS_TYPE_AGGREGATION,
		ObjectFieldConstants.BUSINESS_TYPE_ATTACHMENT,
		ObjectFieldConstants.BUSINESS_TYPE_FORMULA,
		ObjectFieldConstants.BUSINESS_TYPE_RICH_TEXT);
```

and then, just validate as follows:
```
if (_unsupportedBusinessTypes.contains(objectField.getBusinessType())) {
	return null;
}
```

Reply:
Well, I decided to use an enum instead of a Set or another Collection because in my mind we will have a finite number of unsupported business types. For me, finite possibilities make more sense in an enum instead of in a Collection.
An example could be:
* I would create an Enum for the days of the week, the planets of the solar system, etc
* I would create a Collection to store ids, dates, etc

As I thought in this way, I preferred to have that enum in object-rest-api so the whole REST layer can be aware of what business types are supported instead of having it only in the EntityModel.